samples/central-login/README.md

```diff
@@ -0,0 +1,39 @@
+# Central login
+
+In cases in which a centralized login application is desired, in contrast to an embedded login form, the SDK provides a method for leveraging an OAuth flow known as Authorization Code Flow (with PKCE). By using an option (login of "redirect") provided by `TokenManager` to request OAuth/OIDC tokens, when the user is not authenticated, the SDK will redirect the user to the configured login application with the ForgeRock platform. Once authenticated, the user will be redirected back to the original application to attain OAuth/OIDC tokens to complete the user flow.
```

Comment:
> By using an option (login of "redirect") provided by `TokenManager` ...

I feel the above is a bit unclear. Can we be more explicit in what we're describing?

Reply:
Sure, I mostly copy pasted what already existed. This is a good time to edit though.